testsupport/wait/host.go

```diff
@@ -1012,6 +1012,15 @@ func matchNotificationWaitCriterion(actual []toolchainv1alpha1.Notification, cri
 	return true
 }
 
+func matchNotificationWaitCriterion(notification toolchainv1alpha1.Notification, criteria ...NotificationWaitCriterion) bool {
+	for _, c := range criteria {
+		if !c.Match(notification) {
+			return false
+		}
+	}
+	return true
+}
+
```

Comment:
```suggestion
```
We don't really need an additional function since we can just put the single notification in a slice and pass it to the `matchNotificationsWaitCriterion` function.